### ISSUE TITLE
HomeConnect API unexpected values (WTX88RH9GB/01)

### ISSUE THREAD

**Issue by badgertastic** [thoukydides/homebridge-homeconnect#212]
### Plugin Version

0.37.7

### Home Connect Appliance(s)

Bosch Dryer WTX88RH9GB/01

### Log File

```shell
==================================================================================
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] 
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] // Union types
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] export type ProgramKey =
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'LaundryCare.Dryer.Program.Blankets'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Cotton'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Delicates'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.DownFeathers'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Hygiene'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Mix'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Outdoor'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Pillow'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Super40'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Synthetic'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.TimeCold'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.TimeWarm'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.Towels'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Dryer.Program.WoolFinish' // (unrecognised)
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Auto30'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Cotton'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Cotton.Colour'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Cotton.Eco4060'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.DelicatesSilk'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.DrumClean'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.EasyCare'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Mix'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.ShirtsBlouses'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.SportFitness'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Super153045.Super1530'
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]   | 'LaundryCare.Washer.Program.Wool';
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] 
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] // EventNotify
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] export interface EventNotifyValues {
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Option.Duration'?:                   number;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Option.EstimatedTotalProgramTime'?:  number;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Option.FinishInRelative'?:           number;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Option.ProgramProgress'?:            number;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Option.RemainingProgramTime'?:       number;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Root.ActiveProgram'?:                ProgramKey;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'BSH.Common.Root.SelectedProgram'?:              ProgramKey; // (unrecognised)
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect]     'LaundryCare.Dryer.Option.DryingTarget'?:        DryingTarget;
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] }
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] 
[1/5/2024, 9:38:13 AM] [homebridge-homeconnect] ==================================================================================
```

**Comment by thoukydides** [comment]
Thanks for the report.
That extra Dryer program will be in the next release.

**Comment by thoukydides** [comment]
Should be fixed in [v0.37.9](https://github.com/thoukydides/homebridge-homeconnect/releases/tag/v0.37.9).
